aiohttp/client.py

```diff
@@ -167,6 +173,75 @@ def request(self, method, url, *,
 
         return resp
 
+    @asyncio.coroutine
+    def ws_connect(self, url,
+                   protocols=(),
+                   timeout=10.0,
+                   connector=None,
+                   response_class=None,
```

Comment:
How about ws_response_class? Should this be changed in `websocket_client.ws_connect` as well?